packages/create-docusaurus/templates/facebook/docusaurus.config.js

```diff
@@ -148,8 +148,11 @@ const config = {
           alt: 'Meta Open Source Logo',
           // This default includes a positive & negative version, allowing for
           // appropriate use depending on your site's style.
-          src: 'img/meta_opensource_logo_negative.png',
+          src: '/img/meta_opensource_logo_negative.svg',
           href: 'https://opensource.fb.com',
+          style: {
+            maxWidth: '500px',
+          },
```

Comment:
temporary, we can remove it later if we agree on something on the Infima side

Reply:
Why inline style, though? Why not CSS + className?